test/integration/appsource/cymbal_bank_test.go

```diff
@@ -47,12 +47,12 @@ func TestSourceCymbalBank(t *testing.T) {
 
 	region := testutils.GetBptOutputStrSlice(multitenant, "cluster_regions")[0]
 
-	t.Run("AppSource", func(t *testing.T) {
-		t.Parallel()
-		for appName, serviceNames := range testutils.ServicesNames {
-			appSourcePath := fmt.Sprintf("../../../6-appsource/%s", appName)
-			appFactory := tft.NewTFBlueprintTest(t, tft.WithTFDir(fmt.Sprintf("../../../3-appfactory/apps/%s", appName)))
-			for _, serviceName := range serviceNames {
+	for appName, serviceNames := range testutils.ServicesNames {
+		appSourcePath := fmt.Sprintf("../../../6-appsource/%s", appName)
+		appFactory := tft.NewTFBlueprintTest(t, tft.WithTFDir(fmt.Sprintf("../../../3-appfactory/apps/%s", appName)))
+		for _, serviceName := range serviceNames {
+			t.Run(fmt.Sprintf("%s/%s", appName, serviceName), func(t *testing.T) {
```

Comment:
Probably need to capture the range variables

```suggestion
			serviceName := serviceName // capture range variable
			t.Run(fmt.Sprintf("%s/%s", appName, serviceName), func(t *testing.T) {
```